Tensile/KernelWriterAssembly.py

```diff
@@ -12013,7 +12015,9 @@ def globalWriteBatch(self, kernel, ss, batchIdx, applyAlpha, beta, edge, atomic,
             # CEqualsD case, use SrdD instead of SrdC
             LoadCCodeStr = LoadCCodeStr.replace('SrdC', 'SrdD')
           LoadCCodeMod.addCode(LoadCCodeStr)
-
+          
+      if not kernel["BufferStore"] and not kernel["LdcEqualsLdd"]:
```

Comment:
I think we need to use AssertCEqualsD instead of LdcEqualsLdd.
LdcEqualsLdd seems to be deprecated.

Reply:
Ok, we might need to mark as deprecated and reove

Reply:
AssertCEqualsD has a different meaning, that's for when C and D are the same memory buffer. LdcEqualsLdd was just for equal stride values, but if it's not working and no longer needed it should probably just be removed.

Reply:
> Ok, we might need to mark as deprecated and reove

It is already written in the definition of LdcEqualsLdd.

Reply:
> AssertCEqualsD has a different meaning, that's for when C and D are the same memory buffer. LdcEqualsLdd was just for equal stride values, but if it's not working and no longer needed it should probably just be removed.

I understand they are different. CEquelsD implies ldc=ldd and we can do same optimizations.
However, that might not be needed.
We can focus on ldc!=ldd only without considering optimization for ldc=ldd at this moment.